src/eeg.jl

```diff
@@ -70,26 +82,40 @@ function labels2positions(labels)
         if haskey(CHANNEL_TO_POSITION_10_20, key)
             return CHANNEL_TO_POSITION_10_20[key]
         else
-            error("Currently only 10_20 is supported. Found: $(label)")
+            error("Currently only 10_20 is supported. Found label: $(label)")
         end
     end
 end
 
-function Makie.convert_arguments(::Type{<:EEG_TopoPlot}, data::AbstractVector{<: Real})
-    return (data, ["sensor $i" for i in 1:length(data)])
-end
+#function Makie.convert_arguments(::Type{<:EEG_TopoPlot}, data::AbstractVector{<:Real})

```

Comment:
this allowed a user to specify a list of labels instead of coordinates if they were using a subset of 10/20 -- I'm surprised this didn't break tests. Can you add a test where only a subset of 10/20 is plotted by passing labels but not positions?

Reply:
ah I'm seeing the addition below that probably addresses this, but I would still like a test 😄